frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -20,7 +20,30 @@ local C_ = _.pgettext
 local T = require("ffi/util").template
 local Screen = Device.screen
 
-local ReaderHighlight = InputContainer:extend{}
+local ReaderHighlight = InputContainer:extend{
+    long_press_action = {
+        "ask",
+        "nothing",
+        "highlight",
+        "select",
+        "note",
+        "translate",
+        "wikipedia",
+        "dictionary",
+        "search",
+    },
+    long_press_action_text = {
+        _("Ask with popup dialog"),
+        _("Do nothing"),
```

Comment:
Any real reason to separate these value/text mapping into 2 tables that we'll have to not mess their sync ?

Reply:
For the purpose of Dispatcher.

Reply:
OK, I see.
I would have probably made it be `toggle=ReaderHighlight.get_long_press_action_text()`, a function returning the list of text from the old 2-item mappings.

I remember there's been a few discussions about sometime adding functionality for plugins to be able to register some new highlight action to this list (mostly to the button table), although none do that currently. I guess we may have to revisit all that - and it's Dispatcher integration - when the times comes we have to add that :)

Reply:
Done